deployments/prod/environment.py

```diff
@@ -45,7 +45,7 @@ def env() -> Mapping[str, Optional[str]]:
                                      internal=internal,
                                      plugins=dict(metadata=dict(name='hca'),
                                                   repository=dict(name='tdr')))
-                for rel in (8, 1)
+                for rel in (8, 1, 9)
```

Comment:
```suggestion
                for rel in (8, 9, 1)
```

Reply:
While it doesn't affect much other than the order catalogs are given by the `index/catalogs` endpoint, since `dcp4` the precedent has been to add catalogs in ascending order, with `dcp1` (and `lungmap`) at the end.